src/api/telemetry/TelemetryAPI.js

```diff
@@ -730,6 +771,15 @@ export default class TelemetryAPI {
  * @memberof module:openmct.TelemetryAPI~
  */
 
+/**
+ * Provides telemetry staleness data. To subscribe to telemetry stalenes,
+ * new StalenessProvider implementations should be
+ * [registered]{@link module:openmct.TelemetryAPI#addProvider}.
+ *
+ * @interface StalenessProvider
```

Comment:
We should define the functions of a staleness provider here, no?

Reply:
K! let me know if looks ok.